hook_pre-commit.go

```diff
@@ -14,16 +14,16 @@ func HookPreCommitDisable(args []string) error {
 
 func HookPreCommitRun(args []string) error {
 	options := map[string]interface{}{
-		"commit": false,
-		"staged": true,
+		"commit-files": false,
+		"staged-files": true,
 	}
 
 	secrets, err := gs.RunCheck(options)
 	if err != nil {
 		return err
 	}
 	if secrets != 0 {
-		return fmt.Errorf("commit cannot proceed")
+		return cli.NewExitError("commit cannot proceed", 42)
```

Comment:
(I thought 42 _was_ a magic number...)

Agree on following convention and using 1 instead.